README.md

```diff
@@ -220,6 +221,116 @@ name=foo
 
 > When your mouse is over the document link, you can `Ctrl+Click`(`Cmd+Click` for macOS) to open the file in a new tab.
 
+### Tests
+Tests can be added after the __Request Body__ with the `@Tests` delimiter line followed by `JavaScript` tests.
+
+Example of the test delimiter.
+```
+@tests
+```
+
+Tests are defined using the `rc.test` function which takes the test `name` and a `test function`. A passing `test function` executes without error, a failing `test function` throws an error.
+
+Example of an empty test.
+```JavaScript
+@tests
+
+rc.test('this is a test', () => {
+    // Your test funtion body goes here
+});
+
+###
+```
+
+`Test Function` receive a reference to the `request` and the `response` containing the corresponding HTTP request and HTTP response data.
+
+The `request` and `response` objects.
+```JavaScript
```

Comment:
I think it would be clearer to describe the request & response object using typescript
like this - 
```ts
request: {
  method: 'GET' | 'POST' ...,
  url: string ...
}
```

Reply:
@pke
> Very cool. Could this be run from the command line? I guess not, since it depends on VSCode?
> The tests are written directly in the http files?

Right, I do not believe there is a command line option but I do believe it is an open feature request. And yes, the tests are in the .http files, the sample above can be copy/paste into a .http file verbatim.

@Meir017 
Sure, typescript looks more concise. I will look to add better descriptions but I want to avoid trying to capture the universe of all options; like all the HTTP status codes or all the HTTP methods, etc.

Reply:
So for CLI support we would have to extract the engine, that reads and executes http files?
This would also need to handle environments then somehow. So it does not seem like an easy task (without looking at the code base).

Reply:
Saw https://github.com/AnWeber/httpyac/ now, its a CLI for http files. Maybe we could port your test code to this too?